locale/shine/extensions/basecommands/ruRU.json

```diff
@@ -0,0 +1,70 @@
+{
+	"CATEGORY": "Базовые команды управления",
+	"EJECT": "Отстранить от командования",
+	"EJECT_TIP": "Принудительно выкидывает игрока из командной станции или Хайва, без повторной возможности принять командование.",
+	"KICK": "Выкинуть",
+	"KICK_TIP": "Выкинуть игрока с сервера.",
+	"KICK_NO_REASON": "Без причины",
+	"KICK_TROLLING": "Троллинг",
+	"KICK_LANGUAGE": "Мат",
+	"KICK_MIC_SPAM": "Спам в микрофон",
+	"GAG": "Молчанка",
+	"GAG_TIP": "Не даёт возможности игроку использовать голосовой и обычный чат.",
+	"GAG_UNTIL_MAP_CHANGE": "До смены карты",
+	"UNGAG": "Снять молчанку",
+	"UNGAG_TIP": "Возвращает возможности игроку использовать голосовой и обычный чат.",
+	"FORCE_RANDOM": "Случайное распределение",
+	"FORCE_RANDOM_TIP": "Принудительно перемещает игрока(ов) в случайную команду.",
+	"READY_ROOM": "Комната ожидания",
+	"READY_ROOM_TIP": "Принудительно перемещает игрока(ов) в комнату ожидания.",
+	"SET_TEAM": "Задать команду",
+	"SET_TEAM_TIP": "Принудительно перемещает игрока(ов) в выбранную команду.",
+	"CHANGE_MAP": "Смена карты",
+	"CHANGE_MAP_TIP": "Моментально меняет карту.",
+	"CALL_VOTE": "Голосование выбора карты",
+	"CALL_VOTE_TIP": "Запускает голосование выбора карты.",
+	"PLUGIN": "Плагин",
+	"STATE": "Состояние",
+	"LOAD_PLUGIN": "Загрузить плагин",
+	"UNLOAD_PLUGIN": "Выгрузить плагин",
+	"UNLOAD_PLUGIN_TIP": "Временно выгружает плагин.",
+	"UNLOAD_PLUGIN_SAVE_TIP": "Сохранить в настройках отключение плагина.",
+	"LOAD_PLUGIN_TIP": "Временно загружает плагин.",
+	"LOAD_PLUGIN_SAVE_TIP": "Сохранить в настройках включение плагина.",
+	"RELOAD_PLUGIN": "Перезагрузка плагина",
+	"ALLTALK_ENABLED": "Включить межкомандное общение.",
+	"ALLTALK_DISABLED": "Выключить межкомандное общение.",
+	"CHEATS_TOGGLED": "{Enabled:BoolToPhrase:enabled|disabled} читы.",
+	"ALL_TALK_TAG": "[Межкомандное общение]",
+	"ALLTALK_TOGGLED": "{Enabled:BoolToPhrase:enabled|disabled} межкомандное общение.",
+	"ALLTALK_PREGAME_TOGGLED": "{Enabled:BoolToPhrase:enabled|disabled} межкомандное общение в режиме пред-игры.",
+	"ALLTALK_LOCAL_TOGGLED": "{Enabled:BoolToPhrase:enabled|disabled} локальное межкомандное общение",
```

Comment:
Similar to above, this will display some English. `{Enable:BoolToPhrase:enabled|disabled}` will display `enabled` if the passed in value is true, or `disabled` if false. So you'll need to replace the `enabled|disabled` bit.